public_html/index.php

```diff
@@ -71,8 +71,8 @@
         <div class="col-md-4 mb-5">
           <div class="card">
             <div class="card-body">
-              <h5 class="card-title">Documentation</h5>
-              <img class="float-right ml-2" height="100px" src="assets/img/docs.svg" />
+              <h5 class="card-title text-left">Documentation</h5>
```

Comment:
Better to edit the CSS I think (it's left by default): https://github.com/nf-core/nf-co.re/blob/0b1b8d3cdf89040aa37e57d21852bff34006261d/public_html/assets/css/nf-core.css#L501-L503